PovioKit/Classes/Extensions/Foundation/Collection+Povio.swift

```diff
@@ -22,15 +22,16 @@ public extension Collection {
   }
 }
 
-public extension MutableCollection {
-  /// Allows setting or getting an element withed checked index bounds
-  subscript(safe index: Index) -> Iterator.Element? {
-    get {
-      return indices.contains(index) ? self[index] : nil
-    }
-    set {
-      guard let newValue = newValue, indices.contains(index) else { return }
-      self[index] = newValue
-    }
-  }
-}
+// NOTE: disabled due to the race condition issue
+//public extension MutableCollection {
+//  /// Allows setting or getting an element withed checked index bounds
+//  subscript(safe index: Index) -> Iterator.Element? {
+//    get {
+//      return indices.contains(index) ? self[index] : nil
+//    }
+//    set {
+//      guard let newValue = newValue, indices.contains(index) else { return }
+//      self[index] = newValue
+//    }
+//  }
+//}
```

Comment:
@nejcvivod I needed to disable this extension because when enabled, the app crashes. Not really sure why this happens though 🤔